public/v4/logos/baikal.png:Zone.Identifier

```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]

```

Comment:
Added by mistake I assume? :) 
I'll merge this as is and will remove this file myself.